pkg/sysctl/sysctl.go

```diff
@@ -31,6 +31,7 @@ import (
 )
 
 const (
+	// Subsystem is the logging subsystem.
 	Subsystem = "sysctl"
```

Comment:
Why is this exported?

Reply:
A historical leftover, I guess :) I've un-exported the symbol.